canu/generate/switch/config/config.py

```diff
@@ -576,6 +577,25 @@ def add_custom_config(custom_config, switch_config, host, switch_os, custom_file
     return custom_config_merge
 
 
+def get_rack_vlans(sls_variables, rack_vlans, destination_rack_list, switch_name):
+    """Get list of VLANs based on rack."""
+    rack_vlans_list = []
+    for cabinets in sls_variables[rack_vlans]:
```

Comment:
If rack_vlans isn't a key found in sls variables this will throw an exception.  Make sure the calling code doesn't allow rack_vlans that aren't in sls_variables ... or catch the exception that will get thrown.